apps/financial-aid/web-osk/src/components/Summary/FormInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Box, Button, Divider, Icon, Text } from '@island.is/island-ui/core'
+import { useRouter } from 'next/router'
+
+interface InfoProps {
+  id: string
+  label: string
+  url: string
+  info?: string
+}
+
+interface Props {
+  info: InfoProps[]
+  error: boolean
+}
+
+const FormInfo = ({ info, error }: Props) => {
+  const router = useRouter()
+
+  return (
+    <>
+      {info.map((item, index) => {
+        const err = error && item.info === undefined
```

Comment:
Why do. we check for error and item.info?

Reply:
If there is an error, I check where it is. And If any of the necessary information(info) is undefined it means there is an error there.